mempool/mempool.go

```diff
@@ -368,7 +383,8 @@ func (mem *Mempool) notifyTxsAvailable() {
 
 // Reap returns a list of transactions currently in the mempool.
 // If maxTxs is -1, there is no cap on the number of returned transactions.
-func (mem *Mempool) Reap(maxTxs int) types.Txs {
+// If maxTxsBytes is -1, there is no cap on the size of returned transactions.
+func (mem *Mempool) Reap(maxTxs, maxTxsBytes int) types.Txs {
```

Comment:
having `maxTxs` is convenient for testing and `/unconfirmed_txs?limit=N` endpoint, but it can be removed still. But then I am not sure what to do with `/unconfirmed_txs?limit=N` endpoint. Should we limit by total bytes?

Reply:
I think we could just make a separate `ReapTxs` function, which should only be used by the endpoint and testing. I don't think that sort of code repitition will hurt us, as long as we don't use it in consensus.

Reply:
Nice idea!